tests/wallet/test_singleton_lifecycle_fast.py

```diff
@@ -49,9 +49,12 @@ def puzzle_for_hash(self, puzzle_hash: bytes32) -> Optional[Program]:
         return self._db.get(puzzle_hash)
 
 
-def from_kwargs(kwargs: Dict[str, Any], key: str, type_info: Any = Any) -> Any:
-    """Raise an exception if `kwargs[key]` is missing or the wrong type"""
-    """for now, we just check that it's present"""
+def from_kwargs(kwargs: Dict[str, Any], key: str) -> Any:
```

Comment:
it looks like the `type_info` parameter is meant to determine the return type. maybe construct it from the value in the dict, or cast it.